Lib9c/Action/UpdateSell.cs

```diff
@@ -23,11 +23,25 @@ namespace Nekoyume.Action
     /// Updated at https://github.com/planetarium/lib9c/pull/1022
     /// </summary>
     [Serializable]
-    [ActionType("update_sell3")]
+    [ActionType("update_sell4")]
     public class UpdateSell : GameAction
     {
         public Address sellerAvatarAddress;
-        public IEnumerable<UpdateSellInfo> updateSellInfos;
+        public IEnumerable<UpdateSellInfo> updateSellInfos
+        {
+            get => _updateSellInfos;
+            set
+            {
+                if (value.Count() > 100)
+                {
+                    throw new ArgumentOutOfRangeException($"{nameof(value)} must be less than or equal 100.");
+                }
```

Comment:
I think that we need to check in the `.Execute()` too because users still can send `update_sell4` action with 100+ arguments through editing the payload by hand.

Reply:
it will throw exception in LoadPlainValue, but I'll fix it.